resources/user-agents/apps/polaris.json

```diff
@@ -12,69 +12,49 @@
         "Comment": "Polaris",
         "Browser": "Polaris",
         "Browser_Maker": "Infraware Inc",
-        "Browser_Type": "Application",
-        "Frames": "true",
-        "IFrames": "true",
-        "Tables": "true",
-        "Cookies": "true",
-        "JavaScript": "true",
-        "CssVersion": "2"
+        "Browser_Type": "Application"
       },
       "children": [
         {
           "match": "LG-* Polaris/5.* MMP/2.*",
           "properties": {
-            "Version": "5.0",
-            "MajorVer": "5",
-            "MinorVer": "0"
+            "Version": "5.0"
           }
         },
         {
           "match": "LG-* Polaris/6.* MMP/2.*",
           "properties": {
-            "Version": "6.0",
-            "MajorVer": "6",
-            "MinorVer": "0"
+            "Version": "6.0"
           }
         },
         {
           "match": "LG-* Polaris/7.* MMP/2.*",
           "properties": {
-            "Version": "7.0",
-            "MajorVer": "7",
-            "MinorVer": "0"
+            "Version": "7.0"
           }
         },
         {
           "match": "Samsung-* Polaris/5.* MMP/2.*",
           "properties": {
-            "Version": "5.0",
-            "MajorVer": "5",
-            "MinorVer": "0"
+            "Version": "5.0"
           }
         },
         {
           "match": "Samsung-* Polaris/6.* MMP/2.*",
           "properties": {
-            "Version": "6.0",
-            "MajorVer": "6",
-            "MinorVer": "0"
+            "Version": "6.0"
           }
         },
         {
           "match": "Samsung-* Polaris/7.* MMP/2.*",
           "properties": {
-            "Version": "7.0",
-            "MajorVer": "7",
-            "MinorVer": "0"
+            "Version": "7.0"
           }
         },
         {
           "match": "Mozilla/4.0(compatible;Polaris 6.2*;Brew 3.1*)/* Samsung SCH-U820",
           "properties": {
-            "Version": "6.2",
-            "MajorVer": "6",
-            "MinorVer": "2"
+            "Version": "6.2"
```

Comment:
Could we use the #DEVICE# instead with Samsung SCH-U820 and LGE VX8575 ?

Reply:
The `#DEVICE#` token is relatively new (earlier this year) and I've converted a lot of the files to use it, but I haven't touched all of them yet (tried doing the ones with the most cases of it first).  I think we should with these two patterns, but probably doesn't need to be part of this commit.

Reply:
I agree, that's why I made it a comment and not a requirement.
It is also the case for the other comments on versions.